### ISSUE TITLE
How to set up action_sim for fine-tune code?

### ISSUE THREAD

**Issue by zshanggu** [octo-models/octo#134]
Hi, I am running fine-tune code `python scripts/finetune.py --config.pretrained_path=hf://rail-berkeley/octo-small-1.5`

And I'm trying to use aloha dataset (bi-manual), however, I don't find corresponding action config in the config file. For bi-manual, the action dim should be 14, but I always get this error, seems the current action dim is still 7:

<img width="902" alt="Screenshot 2024-10-05 at 20 09 29" src="https://github.com/user-attachments/assets/6cfc6db3-1b36-40c3-934a-b74d7eb0af30">
